setup.py

```diff
@@ -25,5 +25,7 @@
     install_requires=REQUIREMENTS,
     package_data={
         "raymon": ["frontend/*"],
+        "raymon": ["raymon/models/*"],
+        "raymon": ["raymon/tests/profiling/retinopathy_data/3*"],
```

Comment:
You are overwriting the dict key *twice*. Append extra values to the list. Now, the first 2 values will get lost. 

This should cause at least one test to fail though...  So maybe we can write a test for it?